apps/qubit/modules/informationobject/templates/_renameModal.php

```diff
@@ -13,29 +13,27 @@
     <div class="alert"><?php echo __('Use this interface to update the description title, slug (permalink), and/or digital object filename.') ?></div>
 
     <div>
-      <div style="float:right"><input id="renameModalEnableTitle" type="checkbox" checked="checked" /> <?php echo __('Update title') ?></div>
-      <label><?php echo __('Description title') ?></label>
-      <input id="renameModalTitle" name="title" type="text" value="<?php echo $resource->title ?>" />
-      <div class="description">
-        <?php echo __('Editing the description title will automatically update the slug field if the "Update slug" checkbox is selected - you can still edit it after.') ?>
-      </div>
+      <div style="float:right"><input id="rename_enable_title" type="checkbox" checked="checked" /> <?php echo __('Update title') ?></div>
+      <label><?php echo $renameForm->title->label ?></label>
+      <?php echo $renameForm->title ?>
+      <div class="description"><?php echo $renameForm->title->help ?></div>
```

Comment:
Hi Mike.

The label must be related to the input using the "for" and "id" attributes. There is a couple of easy ways to do it, in this three fields I think you can replace the three lines of each field for one of this two options (depending on how translations are handled in this feature):

<?php echo render_field($renameForm->title), $resource) ?> (with translations)
<?php echo $renameForm->title->renderRow() ?> (without translations)